main/read.c

```diff
@@ -135,6 +135,13 @@ extern unsigned long getInputLineNumber (void)
 	return File.input.lineNumber;
 }
 
+extern int getInputLineOffset (void)
+{
+	unsigned char *base = (unsigned char *) vStringValue (File.line);
+	int ret = File.currentLine - base - File.ungetchIdx;
+	return ret >= 0 ? ret : 0;
```

Comment:
Agreed it's unlikely to really matter (even more so as it's likely to be 1 byte at most), but we never know until it bites :)

Anyway, not a blocker, it's a mere comment on a current limitation.